normandy/control/static/control/js/components/RecipeForm.jsx

```diff
@@ -42,6 +101,22 @@ class RecipeForm extends React.Component {
             <textarea field={filter_expression} {...filter_expression} />
           </div>
         </div>
+        <div className="row">
+          <div className="fluid-3">
+            <label>Action</label>
+            <select {...action_name} onChange={this.changeAction}>
+              <option>Select an action</option>
+              {
+                availableActions.map(action => {
+                  return (<option key={action.name} value={action.name}>{action.name}</option>)
```

Comment:
The `return` is unnecessary if you use the short-return form of the double-arrow function. Also destructuring can help:

```js
availableActions.map(({name}) => (
  <option key={name} value={name}>{name}</option>
))
```